packages/ui/src/components/va-time-picker/components/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -139,6 +141,14 @@ export default defineComponent({
     height: 100%;
     border-right: var(--va-time-picker-column-border-right);
 
+    &::before,
```

Comment:
If we want top/center position it can be made in CSS. Before height just need to be set to cellHeight.